deployment/kube-queue-ctrl.yaml

```diff
@@ -0,0 +1,40 @@
+apiVersion: apps/v1beta1
+kind: Deployment
+metadata:
+  labels:
+    component: controller
+    tier: control-plane
+  name: kube-queue-ctrl
+  namespace: kube-system
+spec:
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        component: controller
+        tier: control-plane
+        version: second
+    spec:
+      containers:
+      - command:
+        - /bin/sh
+        - -c
+        - touch /tmp/healthy; /opt/kube-queue-ctrl --scheduler-name kube-batchd; rm -f /tmp/healthy
+        image: kubearbitrator/queue-ctrl:v0.1
```

Comment:
we should not use this repos; it's changed to `kubesigs/kube-batch`

Reply:
And where is the dockerfile build this images? we already move them out.

Reply:
Removed from PR and will make appropriate changes when re-adding to a future PR.